components/StoryStickyBar/StoryStickyBar.tsx

```diff
@@ -26,7 +26,7 @@ function StoryStickyBar({ story }: PropsWithChildren<Props>) {
                     <p className="flex items-center">
                         Powered by
                         <a
-                            href="https://www.prezly.com/"
+                            href="https://prez.ly/storytelling-platform"
```

Comment:
I find it particularly weird on this theme, that there are effectively two "Powered by Prezly" badges on a single screen 😕 
Good to discuss with designer, but IMO this is too much and we should probably leave just one.

<img width="1922" alt="image" src="https://github.com/prezly/theme-nextjs-marcel/assets/1641218/48f704b4-64f6-40ba-8273-4953da991c42">

Reply:
Dropped old link